css/homepage.css

```diff
@@ -0,0 +1,99 @@
+body {
+  font-family: "Courier New", Courier, monospace;
+  margin:0px auto;
+  padding:0px auto;
+  background: #333; /* Grey for darkening */
+}
+h1 {
+  border-bottom: solid;
+  border-bottom-color: var(--highlight);
+    font-weight: lighter;
+}
+h2 {
+  position:relative;
+  vertical-align: middle;
+  line-height:100vh;
+  color:var(--light-text);
+  margin:0px;
+  font-weight: lighter;
+}
+a {
+  margin:0px auto;
+  padding:0px auto;
+}
+
+/*a weird fix for a floating hyperlink on each column*/
+span {
+  position:absolute;
+  width:100%;
+  height:100%;
+  top:0;
+  left: 0;
+  z-index: 1;
+  background-image: url("../img/empty.gif");
+}
+
+.wrap {
+  display:table;
+  width:100vw;
+  height:100vh;
+  margin:0px auto;
+  padding:0px;
+  position:relative;
+  font-size:18px;
+  table-layout:fixed;
+  text-align:center;
+  border-collapse:collapse;
+  overflow:hidden;
+
```

Comment:
Blank lines within braces should probably be removed.
Similarly to the above, inconsistent blank lines between objects. Consistency is a good habit

Reply:
Fixed in https://github.com/Zenloth/zenloth.github.io/commit/fa738271734e8a5b824f82bce319ed7403a7b7aa